test/components/TestHome.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <q-page
+    class="column q-col-gutter-md"
+    padding>
+    <div class="column items-center">
+      <credential-switch
+        :credential="alumniCredential"
+        :clickable="true" />
+      <credential-switch
+        style="max-width: 250px;"
+        :credential="alumniCredential"
+        :expandable="true" />
+    </div>
+  </q-page>
+</template>
+<script>
+/*!
+ * Copyright (c) 2018-2022 Digital Bazaar, Inc. All rights reserved.
+ */
+import {
+  CredentialSwitch
+} from '@bedrock/vue-vc';
+import {alumniCredential} from './credentials';
+const issuer = 'did:v1:test:1234';
```

Comment:
Space after imports section:

```suggestion
import {alumniCredential} from './credentials';

const issuer = 'did:v1:test:1234';
```